src/content-script/eip6963.ts

```diff
@@ -15,19 +15,33 @@ const info = {
   rdns: 'io.zerion.wallet',
 } satisfies EIP6963ProviderInfo;
 
-type Options = { onRequestProvider: () => void };
+type Options = {
+  onRequestProvider?: () => void;
+  onAccessProvider?: () => void;
+};
 
 export function initializeEIP6963(
   provider: EthereumProvider,
   options?: Options
```

Comment:
Do we need these options to be optional?

Reply:
I think it's a good practice, why not?